app/controllers/comments_controller.rb

```diff
@@ -5,10 +5,14 @@ def create
     @comment.wish = Wish.find(params[:wish_id])
     @wish = @comment.wish
     authorize @comment
-    if @comment.save
-      redirect_to wish_path(@comment.wish)
-    else
-      render 'wishes/show', status: :unprocessable_entity
+
+    respond_to do |format|
```

Comment:
There is a simpler way to do this, isn't it? I think you don't need to create a stimulus controller. Instead:

`redirect_to wish_path(@comment.wish, anchor: "comment-#{@comment.id}")`

Reply:
update the order of comments based on date